scene_synchronizer.cpp

```diff
@@ -3479,6 +3513,63 @@ void ClientSynchronizer::process_paused_controller_recovery(real_t p_delta) {
 	scene_synchronizer->change_events_flush();
 }
 
+void ClientSynchronizer::__pcr__sync_no_rewind(const LocalVector<NetUtility::PostponedRecover> &p_postponed_recover) {
```

Comment:
under what conditions does this one occur? Seems that this gets processed for nodes that where the `compare_vars` function returned false and the returned vars array is non-empty in the `pre_rewind` function above.

Reply:
Not all the properties should trigger a rewind. Sometimes there are properties that are useful but doesn't have an impact on the gameplay: take as an example the Nickname, The coin count, etc... for all these properties you can use ` void set_skip_rewinding(node: Node, variable: String, skip_rewinding: bool)` to mark those as no rewinding.

If the scene synchronizer detects that only those properties were different, the rewind is skipped and the properties are just set to the server provided value.